src/__init__.py

```diff
@@ -0,0 +1,52 @@
+# https://py.checkio.org/ (all tasks)
+# 1.Multiply: multiply 2 numbers
+# 2.Easy Unpack: return 1st, 3rd and second to last element from tuple
+# 3.Index Power:Enter N. Raise Nth element of the array to the Nth power,
+# if x[N] doesn't exists then return (-1)
+# 4.Digits Manipulation:Multiply all but 0 digits of positive integers.
+# 5.Secret Message:Gather all capital letters in one word
+# in the order that they appear in the chunk of text.
+
+
+# PALINDROME
+print('input number')
+number = int(input())
+original_num = number
+ostatok = 0
+reversd_num = 0
+while number != 0:
```

Comment:
while num?

Reply:
Maxim,
I would like to discuss some issues regarding HW2. First of all, your comment about adding task from suggested sites was sent after the deadline of 01.11.2019. So, I considered my hw being OK and checking my email for the rest of the time optional. That's why I didn't see the comment mentioned and didn't answer it. Second, I've done all the five tasks needed (see screenshot). And third, it would be very interesting for me to discuss the views on naming variables because I consider it inconvenient to name variables as full russian words since we work in the community of people who know English language and professional programmer lingo.
Best regards,
Nikolai

![proofhw2](https://user-images.githubusercontent.com/56649983/68540514-6a3b1c00-03a4-11ea-970f-1594bcc1ff1d.png)